test/land/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+# test for creating pseudo ensemble for use in letkfoi
+add_test(NAME test_gdasapp_land_create_ens
+         COMMAND ${PROJECT_SOURCE_DIR}/test/land/create_bkg_ens.sh ${PROJECT_BINARY_DIR} ${PROJECT_SOURCE_DIR}
+         WORKING_DIRECTORY ${PROJECT_BINARY_DIR}/test/land/)
```

Comment:
GDASAPP_TESTDATA
Look at other `ctests` @jiaruidong2017 , but you probably need to do something like this:
```
set_tests_properties(test_gdasapp_land_create_ens
    PROPERTIES
    ENVIRONMENT "GDASAPP_TESTDATA=$ENV{GDASAPP_TESTDATA}")
```
make sure that `GDASAPP_TESTDATA` is actually defined by printing it using cmake `message` function ... or something.

Reply:
Thanks @guillaumevernieres and I will try this.